src/__tests__/expected/selectTwoCommandMode.txt

```diff
@@ -0,0 +1,30 @@
+
+
+
+
+
+
+
+
+==========================================================
+Files you have selected:
+==========================================================
+./README.md
+/Users/pcottle/Dropbox (Facebook)/wip/PathPicker/src/__tests__/__init__.py
```

Comment:
oh wait, this wont work outside of my directory because of the relative imports. let me update...